tools/applicationinsights-web-snippet/createAiSnippet.js

```diff
@@ -0,0 +1,39 @@
+// createFile.js
+const fs = require('fs');
+const path = require('path');
+
+// Define the path to the snippet.js file
+const filePath = path.join(__dirname, 'src/snippet.ts'); // Adjust the path if necessary
+const aiFilePath = path.join(__dirname, 'src/aiSnippet.ts');
```

Comment:
It might also be worth either
- Placing these into a different folder (so we know they are generated)
- Explicitly adding these to .gitignore (or a generated folder from above) so that we don't check in these files

Lets also "Add" a comment to the top of the file 
```ts
// DO NOT EDIT - Automatically Generated
```